grails-app/services/org/pih/warehouse/replenishment/ReplenishmentService.groovy

```diff
@@ -42,8 +42,17 @@ class ReplenishmentService {
 
     def getRequirements(Location location, InventoryLevelStatus inventoryLevelStatus) {
         def requirements = Requirement.createCriteria().list() {
-            eq("location", location)
-            eq("status", inventoryLevelStatus)
+            if (inventoryLevelStatus.toString() == 'BELOW_MAXIMUM') {
+                eq("location", location)
+                or {
```

Comment:
Change this entire `or` into:
`'in'("status", inventoryLevelStatus.listReplenishmentOptions())`